app/screens/send-bitcoin-screen/send-bitcoin-completed-screen.tsx

```diff
@@ -92,26 +106,43 @@ const SendBitcoinSuccessScreen: React.FC<Props> = ({ route }) => {
     }
   }, [client, feedbackModalShown, LL, showSuggestionModal, navigation, requestFeedback])
 
+  const MainIcon = () => {
+    switch (status) {
+      case "SUCCESS":
+        return <GaloyIcon name={"payment-success"} size={128} />
+      case "QUEUED":
+        return <GaloyIcon name={"payment-pending"} size={128} />
+      case "PENDING":
+        return <GaloyIcon name={"warning"} color={colors._orange} size={128} />
+    }
+  }
+
+  const SuccessText = () => {
+    switch (status) {
+      case "SUCCESS":
+        return LL.SendBitcoinScreen.success()
+      case "QUEUED":
+        return LL.TransactionDetailScreen.txNotBroadcast({
+          countdown: formatTimeToMempool(
+            timeToMempool(arrivalAtMempoolEstimate as number),
```

Comment:
@sandipndev I refactored `formatTimeToMempool` and needed to wrap the function with `timeToMempool`. this will fix the `27,123 minutes to confirm`